Symbols explainer post-workshop (#323)
w3c/personalization-semantics: explainers/symbols.md

But there are many different symbol sets in use, and people don't tend to learn more than one.

We propose an approach that allows content authors to mark up the _concepts_ relevant to their content,
so that the appropriate symbol(s) for that concept can be rendered for the user,
using _their chosen symbol set_.

We use the set of concepts maintained by [Blissymbolics Communication International](https://www.blissymbolics.org/) (BCI).
These concepts underpin the Blissymbolics (or "Bliss") language&mdash;though our use of the concepts is strictly for mapping from a concept to the appropriate symbol(s) for the user, and is not grammatical in nature.

We're working closely with BCI on this approach, which is comprised of the following:

* **The use of Bliss Unicode code points to encode a concept.**

  A font then maps these code points into the appropriate symbol for the user.

  The code points are under Unicode Consortium review.

* **A registry of concepts (BCI code point sets) for authors to look up.**

  The W3C AAC Symbol Registry provides the code points used to identify a given concept.

> * **Use of the `<ruby>` element for presentation&mdash;with an attribute to indicate this is a symbol.**

Reviewer: ```suggestion
* **Use of the `<ruby>` element for presentation&mdash;with a `lang=zbl` attribute to indicate this is a symbol.**
```

> * **Use of the `<ruby>` element for presentation&mdash;with a `lang=zbl` attribute to indicate this is a symbol.**

Reviewer 2: ```suggestion
* **Use of the `<ruby>` element for presentation&mdash;with a [`lang=zbl`](https://www.loc.gov/standards/iso639-2/php/code_list.php) attribute to indicate this is a symbol.**
```

## Out of scope

> * Mapping from the concepts to the appropriate symbols for the user, and rendering those symbols (this is left to the UA, or an extension).

Reviewer 2: ```suggestion
```

> 

Reviewer 2: ```suggestion
```

> * Providing an exhaustive list of concepts (the [W3C AAC Symbol Registry](#the-w3c-aac-symbol-registry), described below, aims to do this).

Author: ```suggestion
* Providing a way to define new concepts (the [W3C AAC Symbol Registry](#the-w3c-aac-symbol-registry), described below, provides a list of available concepts).
```
